Automattic/jetpack
# Brief: Add retry event

Add event for retry.

## Proposed changes:

* Add `jetpack_ai_breve_retry` event on retry handle action.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


N/A

## Testing instructions:






* Go to your editor (with content).
* Suggest something.
* Click on retry, it should trigger a event (1x1 gif).
